types/tcp-ping/index.d.ts

```diff
@@ -0,0 +1,41 @@
+// Type definitions for tcp-ping 0.1
+// Project: https://github.com/wesolyromek/tcp-ping/issues
+// Definitions by: JUNG YONG WOO <https://github.com/stegano>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface Option {
+    address?: string;
+    port?: number;
+    attempts?: number;
+    timeout?: number;
+}
+
+export function ping(options: Option,
+              callback: (error: Error, result: {
+                  address: string,
+                  port: number,
+                  attempts: number,
+                  avg: number,
+                  max: number,
+                  min: number,
+                  results: {
+                      seq: number,
+                      time: number,
+                      error: Error
+                  }
```

Comment:
The type of `results` is array, `time` and `error` can be `undefined`
https://github.com/apaszke/tcp-ping/blob/master/ping.js#L65
https://github.com/apaszke/tcp-ping/blob/master/ping.js#L71